### PR TITLE
Add BHNT80 hacks

### PR DESCRIPTION
**Pull request by ligi**
Still some info missing - but at least a start - maybe  @TimDaub @jzilg @lavielle @qaismyname @herrmuellerluedenscheid @johnjohndoe can add more details?

### CODE REVIEW

File: _posts/2021-10-26-no80-0b10G.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+* [taxtool](https://github.com/TimDaub/taxtool) by [Tim Daubenschütz](https://github.com/TimDaub)
+* [bikit](https://github.com/lavielle/bikit) by [LaVielle](https://github.com/lavielle) - **Hack of the Month**
+* Package manager from scratch
+* [git prompt shell](https://github.com/jzilg/shellconfig) by [Johannes Zilg](https://github.com/jzilg/shellconfig)
+* IRoam research notebook
+* Educkio by [Marius Kriegerowski](https://github.com/herrmuellerluedenscheid)
+* Solar house monitor
+* [Nix Haskell](https://github.com/qaismyname/nix-haskell) by [Qaiser Abbasi](https://github.com/qaismyname)
```
Comment (qaismyname) [inline _posts/2021-10-26-no80-0b10G.md:17]: LGTM - ty🙏 it was nice giving the talk btw!

**Review by TimDaub — APPROVED** [review]

**Comment by HerrMuellerluedenscheid** [comment]
Instead of
```
* Educkio by [Marius Kriegerowski](https://github.com/herrmuellerluedenscheid
```
maybe sth more explanatory?
```
* Renewable energy on the local power grid ([educk.io](https://educk.io)) by [Marius Kriegerowski](https://github.com/herrmuellerluedenscheid)
```

* Renewable energy on the local power grid ([educk.io](https://educk.io)) by [Marius Kriegerowski](https://github.com/herrmuellerluedenscheid)

**Comment by ligi** [comment]
@HerrMuellerluedenscheid thanks - changed to this - did not know the details of that one

File: _posts/2021-10-26-no80-0b10G.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Berlin Hack & Tell \#80 - 0b10G
+date: 2021-10-26
+time: '19:30'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/281529520/
+---
+
+* [taxtool](https://github.com/TimDaub/taxtool) by [Tim Daubenschütz](https://github.com/TimDaub)
```
Comment (johnjohndoe) [inline _posts/2021-10-26-no80-0b10G.md:10]: Maybe "tax tool for Ethereum"?

File: _posts/2021-10-26-no80-0b10G.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+* [bikit](https://github.com/lavielle/bikit) by [LaVielle](https://github.com/lavielle) - **Hack of the Month**
+* Package manager from scratch
+* [git prompt shell](https://github.com/jzilg/shellconfig) by [Johannes Zilg](https://github.com/jzilg/shellconfig)
```
Comment (johnjohndoe) [inline _posts/2021-10-26-no80-0b10G.md:13]: Better: "Shell prompt  for bash and zsh"

File: _posts/2021-10-26-no80-0b10G.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+layout: post
+title: Berlin Hack & Tell \#80 - 0b10G
+date: 2021-10-26
+time: '19:30'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/281529520/
+---
+
+* [taxtool](https://github.com/TimDaub/taxtool) by [Tim Daubenschütz](https://github.com/TimDaub)
+* [bikit](https://github.com/lavielle/bikit) by [LaVielle](https://github.com/lavielle) - **Hack of the Month**
```
Comment (johnjohndoe) [inline _posts/2021-10-26-no80-0b10G.md:11]: How about: "Bikit - Bicycle lights, turn signals controlled by MicroPython and Arduino Pico"

**Review by johnjohndoe — CHANGES_REQUESTED** [review]
Here are my two cents.

File: _posts/2021-10-26-no80-0b10G.md (hunk 1 above)
Comment (ligi) [reply _posts/2021-10-26-no80-0b10G.md:17]: thanks for giving the talk!

File: _posts/2021-10-26-no80-0b10G.md (hunk 4 above)
Comment (ligi) [reply _posts/2021-10-26-no80-0b10G.md:11]: changed it to "Bikit - Bicycle lights" - but the other one was too large - we try to keep it shortIsh here

**Review by johnjohndoe — APPROVED** [review]
LGTM :shipit:

**Review by lucasrangit — APPROVED** [review]

**Pull request merged**